tests/integration_tests/performance/test_block_performance.py

```diff
@@ -0,0 +1,541 @@
+            st.consumer.StatisticDef(BW_READ_STDDEV, BW_READ,
+                                     st.function.Stddev),
+            st.consumer.StatisticDef(
+                SLAT_READ_MEAN, SLAT_READ, st.function.Avg,
+                criteria.EqualWith(baseline_slat_read["target"],
+                                   baseline_slat_read["target"] *
+                                   baseline_slat_read["delta_percentage"] /
+                                   100)),
+            st.consumer.StatisticDef(
+                SLAT_READ_STDDEV, SLAT_READ, st.function.Stddev),
+            st.consumer.StatisticDef(
+                CLAT_READ_MEAN, CLAT_READ, st.function.Avg,
+                criteria.EqualWith(baseline_clat_read["target"],
+                                   baseline_clat_read["target"] *
+                                   baseline_clat_read["delta_percentage"] /
+                                   100)),
+            st.consumer.StatisticDef(
+                CLAT_READ_STDDEV, CLAT_READ, st.function.Stddev),
+            st.consumer.StatisticDef(
+                LAT_READ_MEAN, LAT_READ, st.function.Avg,
+                criteria.EqualWith(
+                    baseline_lat_read["target"],
+                    baseline_lat_read["target"] *
+                    baseline_lat_read["delta_percentage"] / 100)),
+            st.consumer.StatisticDef(LAT_READ_STDDEV, LAT_READ,
+                                     st.function.Stddev)]
+    return [
+        st.consumer.StatisticDef(IOPS_READ_MEAN, IOPS_READ, st.function.Avg),
+        st.consumer.StatisticDef(IOPS_READ_STDDEV, IOPS_READ,
+                                 st.function.Stddev),
+        st.consumer.StatisticDef(BW_READ_MEAN, BW_READ, st.function.Avg),
+        st.consumer.StatisticDef(BW_READ_STDDEV, BW_READ, st.function.Stddev),
+        st.consumer.StatisticDef(SLAT_READ_MEAN, SLAT_READ, st.function.Avg),
+        st.consumer.StatisticDef(SLAT_READ_STDDEV, SLAT_READ,
+                                 st.function.Stddev),
+        st.consumer.StatisticDef(CLAT_READ_MEAN, CLAT_READ, st.function.Avg),
+        st.consumer.StatisticDef(CLAT_READ_STDDEV, CLAT_READ,
+                                 st.function.Stddev),
+        st.consumer.StatisticDef(LAT_READ_MEAN, LAT_READ, st.function.Avg),
+        st.consumer.StatisticDef(LAT_READ_STDDEV, LAT_READ,
+                                 st.function.Stddev)]
+
+
+def get_rw_stats(env_id, fio_id, cpu=None):
+    """Return read/write statistics."""
+    stats = get_read_stats(env_id, fio_id, cpu)
+    if cpu:
+        baseline_bw_write = cpu["baseline_bw_write"][env_id][fio_id]
+        baseline_iops_write = cpu["baseline_iops_write"][env_id][fio_id]
+        baseline_slat_write = cpu["baseline_slat_write"][env_id][fio_id]
+        baseline_clat_write = cpu["baseline_clat_write"][env_id][fio_id]
+        baseline_lat_write = cpu["baseline_lat_write"][env_id][fio_id]
+        stats.extend([
```

Comment:
This block of code is hard to read and understand due to the mix of indentations and multiple elements being extended.